app/views/directives/service-instance-integrations.html

```diff
@@ -0,0 +1,13 @@
+<div ng-if="($ctrl.integrations | size)">
+  <h3 ng-if="$ctrl.showHeader">Integrations</h3>
+  <service-integration
+    ng-repeat="integration in $ctrl.integrations"
+    is-overview="$ctrl.isOverview"
+    integration="integration"
+    ref-api-object="$ctrl.serviceInstance">
+  </service-integration>
+  <p ng-if="!($ctrl.integrations | size)">
```

Comment:
Seems this could never be the case given the ng-if  above on the div. Could we remove it?